helper-cli/src/main/kotlin/common/Utils.kt

```diff
@@ -192,6 +193,30 @@ internal fun OrtResult.fetchScannedSources(id: Identifier): File {
     return Downloader().download(pkg, tempDir).downloadDirectory
 }
 
+/**
+ * Return the processed copyright statements for each package or project contained in this [OrtResult] grouped by
+ * the package and license ID.
+ */
+internal fun OrtResult.getProcessedCopyrightStatements(): Map<Identifier, Map<String, Set<String>>> {
+    val result = mutableMapOf<Identifier, MutableMap<String, MutableSet<String>>>()
+    val processor = CopyrightStatementsProcessor()
+
+    collectLicenseFindings(omitExcluded = true).forEach { (id, findings) ->
+        val copyRightForPackage = result.getOrPut(id, { mutableMapOf() })
+        findings.forEach { (licenseFindings, _) ->
+            val statements = processor.process(licenseFindings.copyrights.map { it.statement }).let {
+                it.processedStatements.keys + it.unprocessedStatements
+            }
+
+            copyRightForPackage
+                .getOrPut(licenseFindings.license, { mutableSetOf() })
+                .addAll(statements)
+        }
+    }
+
+    return result
+}
```

Comment:
Would it make sense to also remove copyright garbage in this function?

Reply:
That would make sense. I would prefer to do this in a separate commit / PR though if that's fine with you.

Reply:
Sure.